benchmark_groups:
    group_1:
        ptp_service: "ompl_ptp_planning"
        cart_service: "ompl_lin_planning"
        ptp_config:
            planner_id: "RRTConnect"
            planning_time: 5.0
            longest_valid_segment_fraction: 0.001
        cart_config:
            eef_step: 0.1
            jump_threshold: 0.0
        num_runs: 1